NetKAN/KSPRC-Textures.netkan

```diff
@@ -0,0 +1,19 @@
+{
+	"spec_version": "v1.2",
+	"name": "KSPRC Textures",
+	"abstract": "Standalone KSPRC Textures. Kinda useless without proper configs. (CC-BY-ND-4.0)",
+	"identifier": "KSPRC-Textures",
+	"download": "https://spacedock.info/content/Proot_4811/KSPRC_-_Renaissance_Compilation_artworks_remake/KSPRC_-_Renaissance_Compilation_artworks_remake-0.7_PreRelease_3.zip",
+	"version": "0.7",
+	"release_status": "stable",
+	"ksp_version_min": "1.3",
+	"ksp_version_max": "1.3.99",
+	"x_netkan_license_ok": true,
```

Comment:
If you're using `x_netkan_license_ok`, you need a `license`

Reply:
It needs a `license`
Add `"license": "CC-BY-ND-4.0",`

Reply:
CC-BY-ND-4.0 isn't in the schema, see #2160

Reply:
but ok i added dat

Reply:
still not working @Olympic1 even after adding author and license